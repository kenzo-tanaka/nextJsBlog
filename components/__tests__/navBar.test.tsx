import React from "react";
import { createRenderer } from "react-test-renderer/shallow";
import Navbar from "../navBar";

const renderer = createRenderer();
test("Navbar", () => {
  renderer.render(<Navbar />);
  const renderedOutput = renderer.getRenderOutput();
  expect(renderedOutput).toMatchSnapshot();
});
