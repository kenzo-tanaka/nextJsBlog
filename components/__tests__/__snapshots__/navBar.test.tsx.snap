// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar 1`] = `
<nav
  className="bg-white border-b border-gray-200"
>
  <div
    className="max-w-4xl mx-auto px-2 sm:px-6 lg:px-8"
  >
    <div
      className="relative flex items-center justify-between h-16"
    >
      <div
        className="flex-1 flex items-center justify-center sm:items-stretch sm:justify-start"
      >
        <div
          className="flex-shrink-0 flex items-center"
        >
          <h1
            className="font-bold font-sans text-xl text-gray-800"
          >
            <a
              href="/"
              onClick={[Function]}
              onMouseEnter={[Function]}
            >
              {{ kenzo }}
            </a>
          </h1>
        </div>
      </div>
      <div
        className="absolute inset-y-0 right-0 flex items-center pr-2 sm:static sm:inset-auto sm:ml-6 sm:pr-0"
      >
        <a
          href="https://github.com/kenzo-tanaka/nextJsBlog"
          target="_blank"
        >
          <div
            style={
              Object {
                "boxSizing": "border-box",
                "display": "inline-block",
                "margin": 0,
                "maxWidth": "100%",
                "overflow": "hidden",
                "position": "relative",
              }
            }
          >
            <div
              style={
                Object {
                  "boxSizing": "border-box",
                  "display": "block",
                  "maxWidth": "100%",
                }
              }
            >
              <img
                alt=""
                aria-hidden={true}
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMjUiIGhlaWdodD0iMjUiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmVyc2lvbj0iMS4xIi8+"
                style={
                  Object {
                    "border": "none",
                    "display": "block",
                    "margin": 0,
                    "maxWidth": "100%",
                    "padding": 0,
                  }
                }
              />
            </div>
            <img
              alt="GitHub Icon"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style={
                Object {
                  "border": "none",
                  "bottom": 0,
                  "boxSizing": "border-box",
                  "display": "block",
                  "height": 0,
                  "left": 0,
                  "margin": "auto",
                  "maxHeight": "100%",
                  "maxWidth": "100%",
                  "minHeight": "100%",
                  "minWidth": "100%",
                  "objectFit": undefined,
                  "objectPosition": undefined,
                  "padding": 0,
                  "position": "absolute",
                  "right": 0,
                  "top": 0,
                  "visibility": "hidden",
                  "width": 0,
                }
              }
            />
          </div>
        </a>
      </div>
    </div>
  </div>
</nav>
`;
